birdcage_backend/app/__init__.py

```diff
@@ -41,9 +41,9 @@ def create_app(init_celery=True):
     if init_celery:
         # Initialize Celery
         app.config['CELERY_BROKER_URL'] = broker_url
-        app.config['CELERY_RESULT_BACKEND'] = result_backend
+        app.config['RESULT_BACKEND'] = result_backend
```

Comment:
Renaming this quiets down a deprecation warning.

Reply:
Magic. Thanks.